app/client/components/helpCentre/helpCentre.tsx

```diff
@@ -24,60 +25,83 @@ const subtitleStyles = css`
   ${headline.small({ fontWeight: "bold" })};
 `;
 
-const HelpCentre = (_: RouteComponentProps) => (
-  <>
-    <SectionHeader title="How can we help you?" />
-    <KnownIssues />
-    <SectionContent>
-      <div
-        css={css`
-          margin-bottom: ${space[24]}px;
-        `}
-      >
-        <h2 css={subtitleStyles}>Most popular topics</h2>
+const HelpCentre = (_: RouteComponentProps) => {
```

Comment:
Just noticed this didn't get attached to the lined I was hoping but I think from the description I think it's clear what I meant. If not let me know though!

Reply:
I think you are referring to what is within the useEffect on this page?

This PR is just to get live chat into production for test purposes, I believe decisions like where it will be placed, what pages will have access to it, how it's styled etc will be made in due course. Though making it a component later on may be a good shout.

Reply:
Yes, I'm referring to both the bit inside the useEffect. But also the bit that adds the button to the page.

It has already been decided that the Live Chat will live in the Help Centre, accessible from any its pages. Even if that changes in the future we would still want to separate it into its own component though and then move the place where we call it from.

Reply:
The card me and David are working on is just trying to test whether the live chat will work on one specific page. Right now we're just trying to test whether there will be a successful connection between an online agent (David) and a user (me) in prod. Doing anything apart from this at the moment is deliberately out of scope.

Reply:
Do you have any further comments regarding this or will it be marked resolved?

Reply:
At this point the code exists where it does solely for the purposes of testing, which I and others have communicated/made clear to you on and off of this PR thread. This does seem to be a matter of your personal preference to the detriment of the speed of development/testing. As we are in the testing phase this code and where it lives will be changed and updated a few times. It is unfortunate to halt progression in this way.

Reply:
Given this change is so you can test with David let’s put a task in this sprint's backlog to refactor this. Once that’s done I’d happy to +1 this.

Reply:
If you would like to add a task for this in the sprint's backlog go for it though it will likely become superfluous quite quickly

Reply:
I've added a task in our backlog to discuss this as a team whenever the right time comes. https://trello.com/c/kJ3rva6P. Right now this code is good-to-go and sprint goal of being able to test Live Chat end-to-end should be prioritised.